9.1/paper.tex

```diff
@@ -856,6 +857,10 @@ \section{Acknowledgments}
 ``Performance tuning of high-order discontinuous Galerkin solvers for
 SuperMUC-NG''.
 
+R.~M.~Kynch was supported by the Engineering and Physical Science Research
+Council (EPSRC) UK through grant EP/K023950/1 while working at Swansea
+University, UK 2013-2015 where the majority of his contribution was
+undertaken.
```

Comment:
A final nit-pick: Please fill in an empty line here.

Reply:
Oops, broke that when resolving the conflict. Fixed now.

Reply:
Excellent.